documentation/_databases/_docker_database_containers/metachron_userManagement/docker-compose-userManagement.yaml

```diff
@@ -0,0 +1,36 @@
+version: "3.7"
+
+services: 
+  db:
+    container_name: metachron-database-userManagement
+    image: mariadb:latest
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: password
+      MYSQL_DATABASE: userManagement
+      MYSQL_USER: user
+      MYSQL_PASSWORD: password
```

Comment:
Use proper passwords and do not load it up on GitHub.
Can be done later for production.